code/game/objects/structures/signs.dm

```diff
@@ -148,4 +148,48 @@
 /obj/structure/sign/directions/evac
 	name = "escape arm"
 	desc = "A direction sign, pointing out which way the escape shuttle dock is."
-	icon_state = "direction_evac"
\ No newline at end of file
+	icon_state = "direction_evac"
+
+/obj/structure/sign/safetyboard //64 by 32 pixels, takes two tiles
+	name = "Station Safety Sign"
+	desc = "Only you can prevent station spaghettification."
+	icon = 'icons/obj/safetyboards.dmi'
+	icon_state = "major_accident"
+	var/value
+	var/record
+	var/record_desc = " The gold trim signifies that we've reached the current record for safety, no pressure!"
+
+/obj/structure/sign/safetyboard/New()
+	while(ticker.current_state < GAME_STATE_PLAYING)
+		sleep(10)
+	if(ticker.persist && !value)
+		value 	= ticker.persist.last_singulo_release
+		record 	= ticker.persist.record_singulo_release
+
+	var/icon/I
+	if(value == 0)
+		I = icon('icons/obj/safetyboards.dmi',"X0-red")
+		overlays += I
+		return
+	else if(value < 10)
+		I = icon('icons/obj/safetyboards.dmi',"X[value]")
+		overlays += I
+	else
+		var/ones = value - round(value/10)*10 //round() is floor() (WHY?)

```

Comment:
All I know is that I was using it under the assumption that all round() was round() for years.

I never thought I'd have to look at the help file for friggen round()